services/task-svc/migrations/000011_init.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE task_templates
+	ADD organization_id uuid NOT NULL;
```

Comment:
if not null, then we should add a DEFAULT for existing rows

Reply:
what do you suggest as a default, the 0 uuid?

Reply:
If there is no reasonable default, it can not be NOT NULL. We can keep it like it is, if there is no data on staging yet tbh

Reply:
solved, the 0 uuid can be entered as default with a string